### ISSUE TITLE
Plugin Error

### ISSUE THREAD

**Issue by xXChrome7Xx** [TheStonedTurtle/banked-experience#149]
Issue : Plugin not working as intended in RuneLite Client  

- Plugin working as intended last night
- log in this morning and plugin icon is not found on right hand side of client window
- cannot find plugin in downloaded plugins
- search for plugin in community plugins
- Find plugin only to see that "Plugin is incompatible and requires its author to update" (with caution icon next to 2.5)
- file ticket incase was unaware

![Screenshot 2024-04-17 110220](https://github.com/TheStonedTurtle/banked-experience/assets/167332860/3ab623c4-6657-464f-9198-a673262fc836)

Cheers!
- GIM XchromeX

**Comment by TheStonedTurtle** [comment]
A RuneLite change broke the plugin, a fix has been made and we're currently waiting on RuneLite to approve the new Pull Request. You will see this in the client again as soon as they've merged it.
